### PR TITLE
Turn off test verbose mode and give test_requirements

### PR DESCRIPTION
**Pull request by oyvindeide**
Turns off test verbose mode and introduces test_requirements.txt

### CODE REVIEW

**Comment by markusdregi** [comment]
> Turn off test verbose mode

The commit does more then this. Please move the test-requirements.txt (good initiative :+1:) into a separate commit.

**Comment by oyvindeide** [comment]
> > Turn off test verbose mode
> 
> The commit does more then this. Please move the test-requirements.txt (good initiative ) into a separate commit.

Good point, split it into two commits

File: test_requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+pytest
+mock
+decorator
+pylint
+black ; python_version > '3.6'
```
Comment (markusdregi) [inline test_requirements.txt]: Are you sure? Black is 3.6 compatible, so this could be `python_version >= '3.6'`

**Review by markusdregi — APPROVED** [review]
👍

**Pull request merged**